src/services/has-many-getter.js

```diff
@@ -38,20 +46,20 @@ function HasManyGetter(model, association, opts, params) {
         as: params.associationName,
         scope: false,
         required: false,
-        where,
+        where: getWhere(),
         include,
       }],
     })
       .then((record) => ((record && record[params.associationName]) || []));
   }
 
-  function getCount() {
+  async function getCount() {
```

Comment:
[SCARED] You are returning a promise of promise (wait for the count value and return it directly instead of returning a promise of the value) and you don't wait for the 'getWhere()' promise value here again. The `where` operator expects an object.

Reply:
A promise of a promise is a promise 🤔